### PR TITLE
[feature](ddl) Support CREATE DB with COMMENT

### PR DESCRIPTION
**Pull request by xiabaike**
When create a database, add support for comment, which explains something about the database so that people who see it know what it is and what it does. CREATE DATABASE [IF NOT EXISTS] db_name COMMENT database_comment [PROPERTIES ("key"="value", ...)]; Add database's comment to the result of the SHOW CREATE DATABASE statement.

### CODE REVIEW

File: be/src/clucene
Comment (morrySnow) [inline be/src/clucene:1]: should not modify this file

File: fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java [hunk 1]
```diff
@@ -87,8 +87,11 @@ public final class FeMetaVersion {
     // For transaction insert
     public static final int VERSION_132 = 132;
 
+    // support db comment
+    public static final int VERSION_133 = 133;
+
     // note: when increment meta version, should assign the latest version to VERSION_CURRENT
-    public static final int VERSION_CURRENT = VERSION_132;
+    public static final int VERSION_CURRENT = VERSION_133;
```
Comment (morrySnow) [inline fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java:94]: reabase to newest master, version_current is version_133, u need to use version_134

File: be/src/clucene
Comment (xiabaike) [reply be/src/clucene:1]: Yes, I've modified it.

File: fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java (hunk 1 above)
Comment (xiabaike) [reply fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java]: Yes, I've modified it. Thanks.

**Review by morrySnow — APPROVED** [review]

**Pull request closed without merging**